docs/how_to/testing.rst

```diff
@@ -95,6 +97,9 @@ it or the context provided to its template::
                 MyPlugin,
                 'en',
             )
-            html = model_instance.render_plugin({})
+            context = {
```

Comment:
Could you update the example to do the following instead:

```python
renderer = ContentRenderer(request=RequestFactory())
html = renderer.render_plugin(model_instance, {})
```

The `plugin.render()` method is no longer the recommended approach and likely will be deprecated.